tools/fix-spec

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# patch given spec file to have the correct version and declare bundled NPM dependencies
+# Usage: fix-spec <spec-file> <version>
+set -eu
+
+spec="$1"
+version="$2"
+
+PROVIDES=$(npm ls --omit dev --package-lock-only --depth 10 |
+    grep -Eo '[^[:space:]]+@[^[:space:]]+' |
+    sort -u |
+    sed 's/^/Provides: bundled(npm(/; s/\(.*\)@/\1)) = /')
```

Comment:
Explainer for reviewers: the weird .* → \1 substitution is so only replace the *last* occurrence of `@`. We have packages like `@patternfly/patternfly`, whose @ we don't want to substitute.

Reply:
I added a comment.